modules/sdk/gradle-plugins-source-formatter/README.markdown

```diff
@@ -1,40 +1,36 @@
 # Source Formatter Gradle Plugin
 
-The plugin allows to format the project files using the Liferay Source Formatter
-tool.
+The Source Formatter Gradle plugin allows you to format project files using the
+Liferay Source Formatter tool.
 
 ## Usage
 
 To use the plugin, include it in your build script:
 
-```gradle
```

Comment:
Good news! Our Markdown converter is not tripped up by the code syntax highlighting *syntax*; it just renders it as code (with no syntax highlighting). We can use the syntax highlighting here in GitHub and even when we create official docs using this Markdown, there won't be an issue with the Markdown conversion.

I'll reestablish the syntax highlighting Markdown before sending this to BChan. Thanks.

... one of these days we'll add syntax highlighting to our converter. :-)